src/events.rs

```diff
@@ -60,7 +68,6 @@ pub enum Event {
     /// The parameter is true if app was suspended, and false if it has been resumed.
     Suspended(bool),
```

Comment:
Could we move this variant to the `Event` enum? Also perhaps the `Awakened` variant (although I noticed it looks like you're planning on removing this).

I'm wondering if also the `ReceivedCharacter` and `KeyboardInput` variants should be moved there also? Or perhaps they should stay in the `WindowEvent` enum and only be delivered to the currently focused window?